vulnerabilities/importers/redhat.py

```diff
@@ -75,21 +78,53 @@ def to_advisory(advisory_data):
     references = []
     if advisory_data.get("bugzilla"):
         bugzilla = advisory_data.get("bugzilla")
+        url = "https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla)
```

Comment:
In the lines above, this does not make sense and is a risky repeat:
```
    if advisory_data.get("bugzilla"):
        bugzilla = advisory_data.get("bugzilla")
```
... It should be instead:
```
bugzilla = advisory_data.get("bugzilla")
    if bugzilla:
...  
```
No need to fix this just now, yet we will need to get rid of these kind of repetitions.

Reply:
Thanks for this.